crates/ef-testing/src/evm_sequencer/constants.rs

```diff
@@ -41,9 +41,9 @@ lazy_static! {
             block_number: BlockNumber(0),
             block_timestamp: BlockTimestamp(0),
             sequencer_address: *SEQUENCER_ADDRESS,
-            fee_token_address: *FEE_TOKEN_ADDRESS,
+            fee_token_addresses: FeeTokenAddresses{eth_fee_token_address: *ETH_FEE_TOKEN_ADDRESS, strk_fee_token_address: *STRK_FEE_TOKEN_ADDRESS},
```

Comment:
can you add comments explaining these two?

Reply:
This is actually a change made by blockifier to their type, I can but I don't have much context, other then they probably plan for users to be able to pay fees in stark and eth.